api/agent/agent.go

```diff
@@ -1,4 +1,4 @@
-package agent
+package agentpb
```

Comment:
@AlekSi What does it mean `pb`?

Reply:
ProtoBuf. We are slowly moving to the best practices we ignored before: https://github.com/percona/pmm/blob/696407ba1a1c11140a3da6dbf408a53fa0b314eb/api/prototool.yaml#L27 Without it we often have a clash of package names.

Reply:
>And why directory and filename is an agent but package name is agentpb?

That's the next step.